### PR TITLE
Fix return/continue bug, updated readme re model.json, update isObjec…

### PR DESCRIPTION
**Pull request by civsiv**
…t logic in merge function

The numbers are very close but slightly out. The reasons for that are:
- The DB query and script were run about 20 mins apart. Running them at the same time would have caused issues for the Firehose DB.
- Node and Postgres use different geo libraries to calculate distance (though this probably has minimal effect)
- The OO script generates ScheduledSessions from SessionSeries Schedules, and the DB query does not account for this

QA (pre 2 month limit)
SessionSeries Firehose DB count:
Screenshot 2020-10-05 at 17 02 55
SessionSeries OO Script count:
Screenshot 2020-10-05 at 17 03 06

ScheduledSession Firehose DB count:
Screenshot 2020-10-05 at 17 02 20
ScheduledSession OO Script count:
Screenshot 2020-10-05 at 17 03 14

QA (after 2 month limit):
ScheduledSession Firehose DB count:
Screenshot 2020-10-05 at 17 27 57
ScheduledSession OO Script count:
Screenshot 2020-10-05 at 17 27 51

### CODE REVIEW

File: geoSegment/geoSegment.js [hunk 1]
```diff
@@ -269,11 +209,11 @@ async function processSessionSeriesItems(items, segments) {
   for (const item of items) {
     // Filter deleted SessionSeries
     if (item.state === 'deleted') {
-      return;
```
Comment (lukehesluke) [inline geoSegment/geoSegment.js:272]: Great catch!

File: geoSegment/utils/mergeIntoModelExample.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+const { readJsonNullIfNotExists } = require('./readJsonNullIfNotExists');
+
+/**
+ * @param {object} model
+ * @param {object} dataInstance
+ */
+function mergeToModel(model, dataInstance) {
+  // If both are not objects, the object is preferred
+  if (isObject(model) && !isObject(dataInstance)) return model;
+  if (!isObject(model) && isObject(dataInstance)) return dataInstance;
```
Comment (lukehesluke) [inline geoSegment/utils/mergeIntoModelExample.js:14]: you did this thing as well! Thanks!

**Review by lukehesluke — APPROVED** [review]
Looks great Civ! And thanks for filling in the documentation gaps

**Comment by civsiv** [comment]
[Open Objects: Update script to segment files based on geo](https://trello.com/c/4w4KFEau/3113-open-objects-update-script-to-segment-files-based-on-geo)

**Pull request merged**